.idea/kotlinScripting.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="KotlinScriptingSettings" supportWarning="false">
-    <scriptDefinition className="org.jetbrains.kotlin.scripting.resolve.KotlinScriptDefinitionFromAnnotatedTemplate" definitionName="KotlinBuildScript">
-      <order>2147483647</order>
-    </scriptDefinition>
-    <scriptDefinition className="systems.danger.kts.DangerFileScript" definitionName="DangerFileScript">
-      <order>2147483647</order>
-    </scriptDefinition>
-  </component>
-</project>
```

Comment:
This file should not be removed, its important for getting code completion to work in the Dangerfile that is used to automate housekeeping checks.

Reply:
This file is still deleted and needs to be added back